pipeline/src/graph-queries/eventDocuments.ts

```diff
@@ -61,6 +61,16 @@ const query = `{
     series {
       series {
         title
+        contributors {
```

Comment:
Is this what it related to? 
https://github.com/wellcomecollection/content-api/pull/75#issuecomment-1824118943

Reply:
Ha! no I got the original commit comment wrong (fixed it now) because I confused contributors and series contributors
The EventDocument query gets:
- Contributors, which I left out of the transformed document because it doesn't seem very useful for either display or query
- Series, which now includes Series contributors, so it's aligned with Articles

Reply:
So we're fetching Contributors but not using them afterwards at the moment?

Reply:
Indeed. The idea is the query has pretty much everything we _might_ want to use, we can trim it down once the transformed type is a bit more stable/developed to remove anything we don't actually want in the index

Reply:
Works for me :)